karma-build.conf.js

```diff
@@ -8,6 +8,7 @@
 
 module.exports = function(config) {
   require('./karma-base.conf.js')(config);
+  config.files.push('node_modules/core-js-bundle/index.js');
```

Comment:
why do we need this polyfill?

Reply:
Because to compile in `es2015`, `ZoneAwarePromise` must implement
```
    get[Symbol.toStringTag]() {
      return 'Promise' as any;
    }
```

So we need the `core-js` to pass test in `IE`, and in `es2015/evergreen` test, I removed this `polyfill`.